app/helpers/patient_query_helper.rb

```diff
@@ -149,6 +150,9 @@ def patients_by_linelist(current_user, workflow, tab, jurisdiction)
 
       current_user.patients&.where(isolation: true, purged: false)
     else
+      return current_user.patients&.monitoring_active(true) if tab == :active
+      return current_user.patients&.exposure_symptomatic&.merge(current_user.patients&.isolation_requiring_review) if tab == :priority_review
```

Comment:
I think it might be worth defining new scopes in `patient.rb` for these and using ActiveRecord `or` instead of `merge` that way the computation and memory load is on the database (which also means 1 query instead of 2)

```
scope :global_priority_review, lambda {
  exposure_symptomatic.or(isolation_requiring_review)
}
```

Reply:
Great call. Done https://github.com/SaraAlert/SaraAlert/pull/937/commits/c40cb5f646a5a696b9fddd3b6e23e5d03008cf7c